server/public/pluginapi/license_test.go

```diff
@@ -133,19 +133,6 @@ func TestIsE20LicensedOrDevelopment(t *testing.T) {
 			Features:     &model.Features{FutureFeatures: bToP(true)},
 		}))
 	})
-	t.Run("license with E20 SKU name, disabled future features", func(t *testing.T) {
```

Comment:
We need to double check that E20 is in fact no longer being used in the field

Reply:
Being discussed here - https://hub.mattermost.com/private-core/pl/f8i5qw1zcbycjx73es1zrjz46e we should be good with removing e10 and e20 references.